lumigator/jobs/inference/model_clients.py

```diff
@@ -151,6 +152,51 @@ def _chat_completion(
 class HuggingFaceModelClient(BaseModelClient):
     def __init__(self, config: InferenceJobConfig):
         self._pipeline = pipeline(**config.hf_pipeline.model_dump())
+        self._set_tokenizer_max_length()
+
+    def _set_tokenizer_max_length(self):
+        """Set the tokenizer's model_max_length if it's currently the default very large integer.
+        Checks various possible max_length parameters which varies on model architecture.
+        """
+        config = self._pipeline.model.config
+        logger.info(f"Initial model_max_length {self._pipeline.tokenizer.model_max_length}")
+        # If suitable model_max_length is already available, don't override it
+        if self._pipeline.tokenizer.model_max_length != VERY_LARGE_INTEGER:
+            logger.info(f"Using model_max_length = {self._pipeline.tokenizer.model_max_length} \
```

Comment:
Make it a bit more human readable:

```suggestion
            logger.info(f"Setting the maximum length of input tokens to {self._pipeline.tokenizer.model_max_length} \
```

Reply:
Removed this logging statement in favor of the above.